docs/index.md

```diff
@@ -936,6 +936,18 @@ Define a global variable name. For example, suppose your app deliberately expose
 
 By using this option in conjunction with `--check-leaks`, you can specify a whitelist of known global variables that you _expect_ to leak into global scope.
 
+### `--posix-exit-codes`
```

Comment:
From https://github.com/mochajs/mocha/pull/4989#discussion_r1566158084:

> (which might better be called `--unix-exit-codes` based on the discussion here)

Yeah, the requested behavior has evolved a good bit. I wonder if a better name would be `--standard-exit-codes`? That way the word "standard" is intentionally a little ambiguous, to match how this adheres more to the general community standard of how test runners should exit, rather than the precise posix or unix standards...

This isn't a blocker for me. The long-term plan is to make this the default in a future Mocha version anyway. So I'll happily defer to you & the rest of the team. 😄

Reply:
I've no strong opinion on the naming and can change it if that's needed to get final approval, but I'll leave it as-is for now considering that the gnu docs I referenced mention this kind of exit code is standard for posix systems... lmk if I should change it though.

Reply:
Marking as resolved since there seems to be no strong objection to the naming.